api/signals.py

```diff
@@ -0,0 +1,27 @@
+# Third-Party Imports
+from crequest.middleware import CrequestMiddleware
+from django.db.models.signals import post_delete, post_save
+from django.dispatch import receiver
+
+# App Imports
+from core.models import History
+
+
+@receiver(post_save)
+@receiver(post_delete)
+def track_application_actions(sender, instance, **kwargs):
+    current_request = CrequestMiddleware.get_request()
```

Comment:
Seems the only purpose of this package is to get the current request. We could easily create that with a middle-ware. Thus we won't be needing this package.

Reply:
@kmazi Yeah, I have been working on this today, I will remove the package.